packages/components/drawer/src/index.vue

```diff
@@ -19,7 +19,7 @@
           aria-modal="true"
           aria-labelledby="el-drawer__title"
           :aria-label="title"
-          :class="['el-drawer', direction, customClass]"
+          :class="['el-drawer', direction, visible && 'open', customClass]"
```

Comment:
Can we do

```ts
{
  open: visible,
}
```
instead of this? If visible yields false, it will render `false` to the class.